### PR TITLE
Tentatails

### PR DESCRIPTION
**Pull request by crunchypretzels**
Adds a few new Skrell hairstyles, based on art from Coalf and encouraged by Geeves.
image
Also includes loadout tentacle ties- one per hairstyle with them on the sprite- but I'm having some problems with finding the issue preventing them from showing up on the model. If you can find it, reader, I WILL be grateful.

### CODE REVIEW

**Comment by crunchypretzels** [comment]
also i know the last one in the picture has a missing pixel it's fixed in the PR

**Comment by crunchypretzels** [comment]
!review

**Comment by Geevies** [comment]
https://www.youtube.com/watch?v=z-JRdRXiNv4

File: code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm [hunk 1]
```diff
@@ -107,4 +107,21 @@
...
+/datum/gear/ears/skrell/scrunchy
+	display_name = "tentacle tie selection"
+	path = /obj/item/clothing/ears/skrell/scrunchy
+	whitelisted = list(SPECIES_SKRELL)
+	sort_category = "Xenowear - Skrell"
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/ears/skrell/scrunchy/New()
+	..()
+	var/scrunchy = list()
```
Comment (Geevies) [inline code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm:121]:
```suggestion
	var/list/scrunchy = list()
```

File: code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm [hunk 2]
```diff
@@ -107,4 +107,21 @@
...
+	scrunchy["tentacle tie, seaweed"] = /obj/item/clothing/ears/skrell/scrunchy
+	scrunchy["tentacle tie, driftwood"] = /obj/item/clothing/ears/skrell/scrunchy/driftwood
+	scrunchy["tentacle tie, reef"] = /obj/item/clothing/ears/skrell/scrunchy/reef
+	scrunchy["tentacle tie, hoop"] = /obj/item/clothing/ears/skrell/scrunchy/hoop
+	scrunchy["tentacle tie, backwater"] = /obj/item/clothing/ears/skrell/scrunchy/backwater
+	gear_tweaks += new/datum/gear_tweak/path(scrunchy)
```
Comment (Geevies) [inline code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm:127]:
```suggestion
	gear_tweaks += new /datum/gear_tweak/path(scrunchy)
```

**Review by Geevies — APPROVED** [review]

**Review by Chada-1 — APPROVED** [review]

**Pull request merged**